go.mod

```diff
@@ -23,4 +23,13 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
+replace (
+	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	k8s.io/api => k8s.io/api v0.19.2
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.2
+	k8s.io/apiserver => k8s.io/apiserver v0.19.2
+	k8s.io/client-go => k8s.io/client-go v0.19.2
+	k8s.io/code-generator => k8s.io/code-generator v0.19.2
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
+)
```

Comment:
hmm, is there no way to do this without replace? replace afaik is only meant to be used for development.

Reply:
Its a pattern ive seen in k8s libraries as well https://github.com/kubernetes/apiserver/blob/3492629d2b9bafd6ac5e6cdf7521a5c9d2e84227/go.mod#L53-L58. Open to other ways, though.

Reply:
We could also use `sigs.k8s.io/controller-tools v0.4.1` instead of `v0.5.0`. That's where it seems we run into this unintended dep bumping issue. Would provide same exact features of generating v1 crd. Can try it out locally.